payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressElement.kt

```diff
@@ -109,16 +109,30 @@ open class AddressElement constructor(
             baseElements
         }
 
+        country?.let {
+            billingDetailsMap[IdentifierSpec.Country] = it
+        }
+
         sameAsShipping?.let {
             setRawValue(
```

Comment:
I’m noticing that pressing the checkbox doesn’t change its state anymore. Can you take a look at that?

Reply:
It doesn't change the state if the billing address and shipping address are identical. Does that seem ok to you?

Reply:
Weird idea, but should we disable the checkbox in that case? It’d still be visible. This would show that both addresses are identical, but we don’t have this situation where the UI element doesn’t really respond to interaction. What do you think?

Reply:
I updated the PR, when you edit the text field, it will erase the text field and uncheck the checkbox. If you uncheck the checkbox, the text fields get erased as well. This is the best I could do given the short time.